Code/Editor/EditorFramework/Assets/AssetCurator.h

```diff
@@ -316,9 +316,23 @@ class EZ_EDITORFRAMEWORK_DLL ezAssetCurator
 
   void NeedsReloadResources(const ezUuid& assetGuid);
 
+  void InvalidateAssetsWithTransformState(ezAssetInfo::TransformState state);
+
   ///@}
 
-  void InvalidateAssetsWithTransformState(ezAssetInfo::TransformState state);
+  /// \name Utilities
+  ///@{
+
+  /// \brief Generates one transitive hull for all the dependencies that are enabled. The set will contain dependencies that are reachable via any combination of enabled reference types.
+  void GenerateTransitiveHull(const ezStringView sAssetOrPath, ezSet<ezString>& deps, bool bIncludeTransformDebs = false, bool bIncludeThumbnailDebs = false, bool bIncludePackageDebs = false) const;
+
+  /// \brief Generates one transitive hull for all the dependencies that are enabled. The set will contain dependencies that are reachable via any combination of enabled reference types. As only assets can have dependencies, the inverse hull is always just asset GUIDs.
```

Comment:
The comment is mostly identical to the one above, which doesn't make sense to me.